backend/tests/src/lib/common/constants/assertion.constants.ts

```diff
@@ -0,0 +1,5 @@
+enum AssertionConstants {
```

Comment:
Should be in enums folder, not in constants

Reply:
Fixed